.jshintrc

```diff
@@ -0,0 +1,24 @@
+{
+    "white": true,
+    "trailing": true,
```

Comment:
Yep, recycling an old .jshintrc file. They dropped those checks a while back (which is why we use jscs as well).